oddbird/oddcontrast
# Add social share image and og tags

![](https://source.unsplash.com/featured/?cute,animal&social)

## Description
Adds Open graph tags and a static share image.


## Steps to test/reproduce

Share the deploy preview at https://deploy-preview-98--oddcontrast.netlify.app/, and verify the content is as intended.

https://developers.facebook.com/tools/debug may be useful.


## Show me

Facebook
<img width="701" alt="image" src="https://github.com/oddbird/oddcontrast/assets/167908/20b38a2e-4120-4c1a-b10a-b6a07b4937c7">

Elk (Mastodon client)
<img width="948" alt="image" src="https://github.com/oddbird/oddcontrast/assets/167908/1a20449b-c7f0-47e7-a9e2-230c533bc983">
